lib/py/setup.py

```diff
@@ -7,15 +7,10 @@
     long_description=open("../../README.md").read(),
     long_description_content_type="text/markdown",
     url="https://github.com/reddit/edgecontext",
-    project_urls={
-        "Documentation": "https://reddit-edgecontext.readthedocs.io/",
-    },
+    project_urls={"Documentation": "https://reddit-edgecontext.readthedocs.io/"},
```

Comment:
what's going on here?

Reply:
I ran `make fmt` and this changed. left it as its own commit so it's trivial to revert

Reply:
that's... odd. `make fmt` inside the docker image? or using some external version of `black`? older versions of `black` would do this sorta thing, but for a while now (and definitely in the version in the docker image) it'd allow the collection to be multi-line if there's a trailing comma. 

i just tried `make fmt` in a clean build and there were no changes :/

Reply:
Inside the docker image. I'll revert it, but definitely weird.

Reply:
No changes after the revert. Maybe something went funny while figuring out how to get `make test` working for the go tests with a volume mount.